scripts/forward_metaflow_ports.py

```diff
@@ -110,59 +136,77 @@ def run(include_argo, include_airflow):
-                "airflow",
-                "deployment/airflow-deployment-webserver",
-                8080,
-                True,
+                key="airflow",
+                deployment="deployment/airflow-deployment-webserver",
+                port=8080,
+                is_ui=True,
                 namespace="airflow",
-                scheme='https',
-                output_port=9090
+                scheme="https",
+                output_port=9090,
             )
         )
     try:
         while True:
             for port_forwarder in port_forwarders:
                 port_forwarder.step()
-            time.sleep(30)
+            logger.debug(f"Waiting {SLEEP_TIME} seconds before checking health")
+            time.sleep(SLEEP_TIME)
 
     except KeyboardInterrupt:
-        log("Aborted!")
+        logger.info("Aborted!")
         return 0
     finally:
         for port_forwarder in port_forwarders:
             port_forwarder.stop_port_fwd_proc()
-            log("Terminated port forward to %s" % port_forwarder.key)
+            logger.info(f"Terminated port forward to {port_forwarder.key}")
 
 
 def main():
-    parser = argparse.ArgumentParser(description="Maintain port forwards to Kubernetes Metaflow stack")
-    parser.add_argument('--include-argo', action='store_true',
-                        help="Do port forward for argo server (needed for Argo UI)")
-    parser.add_argument('--include-airflow', action='store_true',
-                        help="Do port forward for argo server (needed for Argo UI)")
+    parser = argparse.ArgumentParser(
+        description="Maintain port forwards to Kubernetes Metaflow stack"
+    )
+    parser.add_argument(
+        "--config-file",
+        default=f"{os.getcwd()}/kubeconfig",
+        help="Location of kubeconfig file for the cluster",
+    )
+    parser.add_argument(
+        "--include-argo",
+        action="store_true",
+        help="Do port forward for argo server (needed for Argo UI)",
+    )
+    parser.add_argument(
+        "--include-airflow",
+        action="store_true",
+        help="Do port forward for airflow server (needed for Airflow UI)",
+    )
+    parser.add_argument("--debug", action="store_true", help="Debug logging")
+    parser.add_argument(
+        "--use-gke-auth",
+        action="store_true",
+        help="Enable GKE auth plugin for GCP environments",
+    )
 
     args = parser.parse_args()
+    if args.debug:
+        logger.setLevel(logging.DEBUG)
+    if args.use_gke_auth:
+        os.environ["USE_GKE_GCLOUD_AUTH_PLUGIN"] = "True"
 
     try:
         subprocess.check_output(["which", "kubectl"])
     except Exception:
         print("kubectl must be installed!")
         return 1
 
-    try:
-        subprocess.check_output(["which", "nc"])
-    except Exception:
-        print("nc utility must be installed!")
-        return 1
-
     return run(args.include_argo, args.include_airflow)
 
 
-if __name__ == '__main__':
```

Comment:
What code formatter is being used for this PR?  I'd suggest we standardize on `black`, which is used also for the main Metaflow project.

If we can make it a [pre-commit](https://pre-commit.com/) hook, that'd be awesome.  That way formatting remains good over the long term.

Does not need to be this PR.